docs/en/index.rst

```diff
@@ -31,6 +31,7 @@ LMDeploy has the following core features:
 
 * **Interactive Inference Mode**: By caching the k/v of attention during multi-round dialogue processes, the engine remembers dialogue history, thus avoiding repetitive processing of historical sessions.
 
+* **Excellent Compatibility**: LMDeploy supports [KV Cache Quant](docs/en/quantization/kv_quant.md), [AWQ](docs/en/quantization/w4a16.md) and [Automatic Prefix Caching](docs/en/inference/turbomind_config.md) to be used simultaneously.
```

Comment:
```text
`KV Cache Quant <https://lmdeploy.readthedocs.io/en/latest/quantization/kv_quant.html>`_, 
`AWQ <https://lmdeploy.readthedocs.io/en/latest/quantization/w4a16.html>`_ 
`Automatic Prefix Caching <https://lmdeploy.readthedocs.io/en/latest/quantization/w4a16.html>`_
```

Reply:
ok

Reply:
not rendered correctly as you see here
https://lmdeploy--1988.org.readthedocs.build/en/1988/

Reply:
ok

Reply:
It works now. Thanks! https://lmdeploy--1988.org.readthedocs.build/en/1988/